regression/contracts/reject_history_expr_in_assigns_clause/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--enforce-contract foo
+^main.c.*error: assigns clause target must be a non-void lvalue or a call to one of __CPROVER_POINTER_OBJECT, __CPROVER_assignable, __CPROVER_object_whole, __CPROVER_object_upto, __CPROVER_object_from$
```

Comment:
Do we still want `__CPROVER_POINTER_OBJECT` here?

Reply:
Yes for compatibility reasons until we merge the new function contrats and loop contracts